css/style.css

```diff
@@ -0,0 +1,84 @@
+body {
+    background-color: whitesmoke;
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+header {
+    padding: 15 25;
+    top: 0;
+    position: fixed;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: whitesmoke;
+    flex-direction: row;
+    width: 100%;
+}
+
+nav {
+    /* width: 30%; */
+}
```

Comment:
commented code must be removed from the codebase

Reply:
- [ ] [Required] You really did well reviewing this code. However, I think you should add a little bit of politeness and be more suggestive rather. An author always expects a reviewer to acknowledge the effort exerted while writing the code by being very polite. Use less command-like comments.
Example:
```
You did well commenting out unwanted code. I think it would be better to remove the comment entirely to make the codebase cleaner. After all, we can always leverage version control systems to get back any piece of code.
```
What do you think about the above comment? You can change your comment to something like the above example.

Reply:
did well commenting out unwanted code. I think it would be better to remove the comment entirely to make the codebase cleaner. After all, we can always leverage version control systems to get back any piece of code.